### DISCUSSION TITLE
Policy to set the default folder for placing new bookmarks

### DISCUSSION THREAD

**Discussion by ineuw** [mozilla/policy-templates#841] [Ideas]
Is there a policy to set the default folder for new bookmarks? 

I know that it can be defined in mozilla.cfg etc. . . .  but from the moment I began using policies, I dropped using .cfg. 

Recent versions of Firefox (94 & 95) were changed and newly added bookmarks use the last used folder, which requires time to find it. This is one of the meanest changes introduced by the developer puppy mill at Mozilla. The previous one was the removal of the home button.

**Comment by mkaply** [comment]
You can set the default folder in the Preferences policy.

https://github.com/mozilla/policy-templates/blob/master/README.md#preferences

The pref is:

browser.bookmarks.defaultLocation

and the possible values are:

toolbar or menu or unfiled

As far as the home button goes, we have a policy to bring that back as well:

https://github.com/mozilla/policy-templates/blob/master/README.md#showhomebutton

and if you configure any homepage via policy (or if the user sets a homepage), the button comes back.

**Comment by ineuw** [comment]
Many thanks for the reply.
